src/deadlock.rs

```diff
@@ -1,4 +1,38 @@
-pub use parking_lot_core::deadlock::{acquire_resource, release_resource};
+//! [Experimental] Deadlock detection
+//!
+//! Enabled via the `deadlock_detection` feature flag.
+//!
+//! # Example
+//!
+//! ```
+//! #![feature(deadlock_detection)]
+//! use std::thread;
+//! use std::time::Duration;
+//! use parking_lot::deadlock;
+//!
+//! thread::spawn(move || {
```

Comment:
Add a comment here: `// Create a background thread which checks for deadlocks every 10s`